core/classifiers/base.py

```diff
@@ -94,8 +95,8 @@ def train(self, training_data):
 
     @abc.abstractmethod
     def validate(self, frozen_model_proto):
-        """Validates frozen model protobuf object containing parameters of
-        trained classifier model.
+        """Validates the specified frozen model protobuf object containing
+        parameters of trained classifier model.
```

Comment:
``parameters of the trained classifier model.``

Reply:
Done